crates/services/p2p/src/service.rs

```diff
@@ -903,19 +1054,58 @@ impl SharedState {
     }
 }
 
-pub fn new_service<V, B>(
+pub fn build_shared_state(
+    config: Config<NotInitialized>,
+) -> (SharedState, Receiver<TaskRequest>) {
+    let (request_sender, request_receiver) = mpsc::channel(1024 * 10);
+    let (tx_broadcast, _) = broadcast::channel(1024 * 10);
+    let (new_tx_subscription_broadcast, _) = broadcast::channel(1024 * 10);
+    let (block_height_broadcast, _) = broadcast::channel(1024 * 10);
```

Comment:
can we move the magic numbers out of this function and define it as -
a. a global, or
b. include it in the arguments to the function?

Reply:
The code was using magic number, idk why it's not in the config so I added a global.